janus_server/src/task.rs

```diff
@@ -588,61 +588,63 @@ pub mod test_util {
         }
     }
 
-    /// Create a dummy [`Task`] from the provided [`TaskId`], with
-    /// dummy values for the other fields. This is pub because it is needed for
-    /// integration tests.
-    pub fn new_dummy_task(task_id: TaskId, vdaf: VdafInstance, role: Role) -> Task {
-        let (aggregator_config_0, aggregator_private_key_0) =
-            generate_test_hpke_config_and_private_key();
-        let (mut aggregator_config_1, aggregator_private_key_1) =
-            generate_test_hpke_config_and_private_key();
-        aggregator_config_1 = HpkeConfig::new(
-            HpkeConfigId::from(1),
-            aggregator_config_1.kem_id(),
-            aggregator_config_1.kdf_id(),
-            aggregator_config_1.aead_id(),
-            aggregator_config_1.public_key().clone(),
-        );
-
-        let vdaf_verify_key = SecretBytes::new(
-            iter::repeat_with(|| thread_rng().gen())
-                .take(vdaf.verify_key_length())
-                .collect(),
-        );
+    impl Task {
```

Comment:
Perhaps the `#[cfg(feature = "test-util")]` from line 347 is meant to go here? Though if this is needed in non-test configs, I don't mind exporting it.

Reply:
Indeed, though in this case it's not needed here directly as the `test_util` module it is placed into already requires the `test-util` feature.